.github/workflows/build.yml

```diff
@@ -169,6 +171,9 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: openfpga
+      - name: Update dockerignore to add openfpga_flow
+        run: |
+          cat .dockerignore
```

Comment:
"Update" implies change, this only dumps, was that the intention?

Reply:
Yup, it should be dump . I will address that in next pull request.
I was initially using this section to update docker image with updated context. But later I moved that to `.dockerignore` directly